libraries/AP_HAL_ChibiOS/hwdef/f303-MatekGPS/hwdef.dat

```diff
@@ -150,6 +150,8 @@ define HAL_PERIPH_ENABLE_AIRSPEED
 define HAL_PERIPH_ENABLE_RANGEFINDER
 define HAL_PERIPH_ENABLE_MSP
 
+define HAL_PERIPH_GPS_PORT_DEFAULT 2
```

Comment:
this should be 3 I think

Reply:
Doh! I was tricked by
SERIAL_ORDER USART1 EMPTY USART3 USART2

3 is what it was before via default defines but this UART3 distracted me. My bad. I just need to remove this define and use the default.

Reply:
yep, thanks!